_docs/tasks/istio-auth.md

```diff
@@ -172,3 +172,62 @@ kubectl exec <app-pod> -c proxy -- cat /etc/envoy/envoy-revX.json | grep ssl_con
 # Expect ssl_context in the output.
 ```
 
+## Playing with auth 
```

Comment:
cc @wattli @myidpt . This seems much more cohesive. Setting up auth and playing with auth. The bookinfo sample setup is cleaner as well.